litmus/AArch64Compile_litmus.ml

```diff
@@ -1204,10 +1204,25 @@ module Make(V:Constant.S)(C:Config) =
 
     let kernel_mode = [{ empty_ins with memo="svc #471"}]
 
-    let fault_handler_prologue p =
-      map_ins ["b 1f";sprintf  "asm_handler%d:" p]
-
-    and fault_handler_epilogue code =
+    let user_prologue =
+      ["mrs %[tr0],esr_el1";
+       "lsr %[tr0],%[tr0],#26";
+       "cmp %[tr0],#21";
+       "cset %[tr0],ne";
+       "lsl %[tr0],%[tr0],#2";
+       "adr %[tr1],2f";
+       "add %[tr1],%[tr1],%[tr0]";
+       "br %[tr1]";
+       "2:";
+       "b 1f";
```

Comment:
Can't we simplify all this to:
`     "b.eq 1f";
`
?

Reply:
Ah I wanted to avoid a conditional branch. In fact this "computed jump" also is a dependency..

Reply:
Yes that's a good point. When I tried something similar I had the same problem.

Reply:
Another solution would be to change the vector table entry before the final `svc` instruction. Some synchronisaton  may be needed. I have no idea which.

Reply:
I think you have to do it at the last invocation of the fault handler (you have to be at EL1), but it might be tricky to know that last invocation is.

Reply:
@relokin I'd rather have  try of this alternative implementation in another PR. This PR looks ready to me. Your opinion?

Reply:
Sounds good. I don't have any other comments :), please feel free to go ahead with this PR.

Reply:
Thanks, just merged.